manifests/m/Microsoft/Azure/CosmosEmulator/2.14.12/Microsoft.Azure.CosmosEmulator.locale.en-US.yaml

```
@@ -0,0 +1,19 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Microsoft.Azure.CosmosEmulator
+PackageVersion: 2.14.12
+PackageLocale: en-US
+Publisher: Microsoft® Corporation
+Author: Microsoft
+PackageName: Azure Cosmos DB Emulator
+PackageUrl: https://docs.microsoft.com/en-us/azure/cosmos-db/local-emulator
+License: Copyright (C) Microsoft Corporation
+LicenseUrl: https://www.microsoft.com/en-us/legal/intellectualproperty/copyright/default.aspx

```

```suggestion
License: Proprietary
Copyright: Copyright (C) Microsoft Corporation
CopyrightUrl: https://www.microsoft.com/en-us/legal/intellectualproperty/copyright/default.aspx
```
A copyright isn't a license